release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -175,6 +175,52 @@ With this release, you can access the NMstate Operator and resources such as the
+==== Standardized STS configuration workflow
+{product-title} 4.14 provides a streamlined and standardized procedure to configure Security Token Service (STS) with the AWS Elastic File Storage (EFS) Container Storage Interface (CSI) Driver Operator.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc#efs-sts_persistent-storage-csi-aws-efs[Configuring AWS EFS CSI Driver Operator with Security Token Service].
+
+[id="ocp-4-14-storage-rwod-access-mode"]
+==== Read Write Only Pod access mode (Technology Preview)
+{product-title} 4.14 introduces a new access mode for persistent volumes (PVs) and persistent volume claims (PVCs) called ReadWriteOncePod (RWOD), which can be used only in a single pod on a single node. This is compared to the existing ReadWriteOnce access mode where a PV or PVC can be used on a single node by many pods. This feature is supported with Technology Preview status.
+
+For more information, see xref:../storage/understanding-persistent-storage.adoc#pv-access-modes_understanding-persistent-storage[Access modes].
+
+[id="ocp-4-14-storage-robust-volmgr"]
+==== Robust VolumeManager reconstruction (Technology Preview)
+This feature changes how the kubelet starts and cleans up orphaned volumes. This change improves how the node and kubelet startup. If you find any issues with this feature, please report them. This feature is supported with Technology Preview status.
+
+[id="ocp-4-14-storage-gcp-filestore-csi-driver"]
+==== GCP Filestore storage CSI Driver Operator is generally available
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Google Compute Platform (GCP) Filestore Storage. The GCP Filestore CSI Driver Operator that manages this driver is now generally available.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc[Google Compute Platform Filestore CSI Driver Operator].
+
+[id="ocp-4-14-storage-automigration-vsphere"]
+==== Automatic CSI migration for VMware vSphere
+Starting with {product-title} 4.8, automatic migration for in-tree volume plugins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. 
+
+Support for vSphere was provided in this feature in {product-title} 4.10 at the Technology Preview level, and vSphere support became generally available in 4.13. However, while for new installations of {product-title}, automatic migration was enabled by default, when upgrading from {product-title} 4.12, or earlier, to 4.13, automatic CSI migration for vSphere only occurred if you opted in. In {product-title} 4.14, CSI migration for vSphere is now enabled by default under all circumstances and requires no action by an administrator.
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users.
```

Comment:
Looks good from a general standpoint. We may have to link to a KCS  that describes the two remaining issues

Reply:
OK. Keeping this comment open for this.